### PR TITLE
clarify that NeXus utilities are not directly code

### PR DESCRIPTION
**Pull request by benajamin**
fixes https://github.com/nexusformat/NIAC/issues/91

### CODE REVIEW

File: manual/source/examples/index.rst [hunk 1]
```diff
@@ -45,12 +45,13 @@ These examples illustrate the use of the NAPI
 
 .. _Examples.readers:
 
-Code that reads NeXus data files
+Code Projects that work with NeXus data files
 ################################
```
Comment (prjemian) [inline manual/source/examples/index.rst:49]: needs more `#` to match length of previous line

File: manual/source/examples/index.rst [hunk 2]
```diff
@@ -45,12 +45,13 @@ These examples illustrate the use of the NAPI
 
 .. _Examples.readers:
 
-Code that reads NeXus data files
+Code Projects that work with NeXus data files
```
Comment (prjemian) [inline manual/source/examples/index.rst:48]: How about `Tools that work with NeXus data files` ?  Seems more consistent with https://github.com/nexusformat/NIAC/issues/91

**Review by prjemian — CHANGES_REQUESTED** [review]
as noted

File: manual/source/examples/index.rst (hunk 2 above)
Comment (benajamin) [reply manual/source/examples/index.rst:48]: Normally, I'd agree, but I see this as a special case. My thinking is that people come to this page looking for code examples. "Tools" doesn't indicate that example code can be found, while "code projects" does imply that one can find example code with a bit of digging.

Comment (prjemian) [reply manual/source/examples/index.rst:48]: Ok

**Review by prjemian — APPROVED** [review]
LGTM

**Pull request merged**